docs/editor_manual/new_pages/the_promote_tab.rst

```diff
@@ -14,3 +18,12 @@ The second, *Promote*, is where you can set all the 'metadata' (data about data!
 
 .. Note::
     You may see more fields than this in your promote tab. These are just the default fields, but you are free to add other fields to this section as necessary.
+
+The Settings Tab
+~~~~~~~~~~~~~~~~
+
+The *Settings* has two fields by default.
+
+* **Go Live date/time:** Sets the time in which the changes should go live when published.  If you publish a page that has this field set to a time in the future it will be scheduled until the time comes.
```

Comment:
time in which -> time at which
I suggest removing the second sentence "If you publish a page that has this field set to a time in the future it will be scheduled until the time comes."